README.md

```diff
@@ -2,31 +2,202 @@
+   `testPerson(employee);`
+2. Press step-into to call the `testPerson` method, passing the `employee` variable as the argument.
+   Pass by value means the value stored in the argument `employee` (a memory address) will be copied
+   into the parameter `p`. As a result, both `employee` and `p` point at the same object.    
+   ![person step into](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step0.png)
+3. Press step-over to execute each statement in the `testPerson` method:
+
+| Code                | Visualizer                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `p.age = 30;`       | ![person step1](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step1.png) <br>The age is changed.                                                                                                                                                                                                                       |
+| `p = new Person();` | ![person step2](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step2.png) <br>Parameter <code>p</code> points to the new object.<br>Variable <code>employee</code>still points to the original object due to pass by value.<br>If Java used pass by reference, <code>employee</code>would also point to the new object. |
+| `p.name = "Jie";`   | ![person step3](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step3.png) <br>The name of the new object is changed.                                                                                                                                                                                                    |
+| `p.age = 50;`       | ![person step4](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step4.png) <br>The age of the new object is changed.                                                                                                                                                                                                     |
+
+
+After returning to the `main` method, we see the `age`  has been updated,
+but `employee` still points to the same `Person` object in memory.  If Java
+implemented pass by reference, the `employee` variable would point to the
```

Comment:
Since Java doesn't use pass-by-reference, part of me wants to say to omit this sentence so students aren't confused?

Reply:
ok, removed.